brave/src/main/java/com/linecorp/armeria/client/brave/ClientRequestContextAdapter.java

```diff
@@ -113,6 +115,7 @@ public long startTimestamp() {
     @SuppressWarnings("ClassNameSameAsAncestorName")
     private static final class HttpClientResponse extends brave.http.HttpClientResponse {
         private final RequestLog log;
+        private brave.http.HttpClientRequest request;
```

Comment:
is being lazy here (and in server) over-optimization from POV of armeria?

Reply:
Think it's good.